### PR TITLE
Design change

### PR DESCRIPTION
**Pull request by AppLoidx**
This PR contains 3 points:
* Added logo for plugin
* Changed property `myPadding` from 0 to 15 as default
* Added code screenshot example in README (without external sources)

### Plugin Icon
Added simple plugin icon which uses [action colors](https://jetbrains.design/intellij/principles/icons/#action-icons) and [specification](https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_icon_file.html) from JetBrains

image

### `myPadding` default property
**was**:

image

**now**:

image

### Examples in README
I hope you will not mind about html tags in markdown, because github supports it, but I'm not sure it will be work on all markdown readers

Screenshots saved in `screenshots` dir

> And sorry about one big PR :)

### CODE REVIEW

**Comment by AppLoidx** [comment]
works on IntelliJ IDEA 2020.2 EAP (Ultimate Edition)

**Comment by amaembo** [comment]
Thank you for your PR.

1. I don't like the idea of default padding. 15 may be ok for your purposes but it's not ok for my purposes. If you paste to the place with matching background color (e.g. presentation slide), you don't need padding at all, it's just a file size waste. Also, the padding size will look different depends on the font size, monitor scale, and the selected scale factor. So, to me, 0 is the only reasonable default. It doesn't take so much time for users to change it if necessary. Please rollback this change.

2. Posted screenshots are too big. Is it really necessary to have 6Mpix image in readme?

File: resources/META-INF/pluginIcon.svg [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	.st1{fill:#FFFFFF;}
+</style>
+<circle class="st0" cx="20.02" cy="20.06" r="16.04"/>
+<g>
+	<path class="st1" d="M16.24,25.06c0.86,0.53,2.11,0.97,3.43,0.97c1.96,0,3.1-1.03,3.1-2.53c0-1.39-0.79-2.18-2.79-2.95
+		c-2.42-0.86-3.92-2.11-3.92-4.2c0-2.31,1.91-4.03,4.8-4.03c1.52,0,2.62,0.35,3.28,0.73l-0.53,1.56c-0.48-0.26-1.47-0.7-2.82-0.7
+		c-2.02,0-2.79,1.21-2.79,2.22c0,1.39,0.9,2.07,2.95,2.86c2.51,0.97,3.79,2.18,3.79,4.36c0,2.29-1.69,4.27-5.19,4.27
+		c-1.43,0-2.99-0.42-3.78-0.95L16.24,25.06z"/>
+</g>
+<g>
```
Comment (amaembo) [inline resources/META-INF/pluginIcon.svg:16]: Why so many empty `<g>` tags?

File: resources/META-INF/pluginIcon.svg [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.1.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Слой_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (amaembo) [inline resources/META-INF/pluginIcon.svg:3]: Why "Слой_1" in Russian?

**Comment by AppLoidx** [comment]
Okay, thank you for your fast reply. If you don't mind about logo - I can make one another PR for it (only logo).

**Comment by amaembo** [comment]
Thanks.

**Pull request merged**